DTIPrep/DTIPrepRegister.pl

```diff
@@ -1009,21 +1155,24 @@ sub getPipelineDate {
 }
 
 
+=pod
 
+=head3 insertReports($minc, $registeredXMLFile, $registeredQCReportFile)
 
+Inserts in the MINC header the path to DTIPrep's QC text and XML reports and XML
```

Comment:
...in the MINC _file's_ header...

Grammar: instead of "Inserts _where_ _what_" I would say "Insert _what_ _where_"

Reply:
done